content/blog/how-to-get-started-with-net/index.md

```diff
@@ -0,0 +1,124 @@
+---
+{
+    title: "How to get started with .NET",
+    description: "Did you know that 35% of developers are using .NET? This is a great article to read to get started with .NET.",
+    published: '2022-01-17T02:20:19.000Z',
+    authors: ['bobrossrtx'],
+    tags: ['dotnet', 'c#'],
+    attached: [],
+    license: 'cc-by-4'
```

Comment:
You mentioned you cross-posted this from elsewhere. That's awesome! That said, we need to make sure that we're using the other site as the canonical reference. Otherwise our SEO values will suffer

```suggestion
    license: 'cc-by-4',
    originalLink: 'https://dev.to/bobrossrtx/how-to-get-started-with-net-50bh'
```